sdk/include/ndk/potypes.h

```diff
@@ -25,6 +25,13 @@ Header Name:
 #include <umtypes.h>
 #ifndef NTOS_MODE_USER
 #include <ntpoapi.h>
+#else
+typedef struct _SYSTEM_POWER_INFORMATION {
```

Comment:
Please put this in its own ifdef, not an else, and give it a heading like:
```
//
// Information Structures for NtPowerInformation
//
```
Also, the brace goes on its own line

Reply:
Done.